default/templates/components/stork/stork-search-js.tpl

```diff
@@ -0,0 +1,45 @@
+<script src="https://files.stork-search.net/releases/v1.5.0/stork.js"></script>
+<ema:metadata>
+	<with var="template">
+		<script data-emanote-base-url="${value:baseUrl}">
+			window.searchShown = false;
+			function toggleSearch() {
+				document.getElementById('stork-search-container').classList.toggle('hidden');
+				window.searchShown = document.body.classList.toggle('stork-overflow-hidden-important');
+				if (window.searchShown) {
+					document.getElementById('stork-search-input').focus();
+				}
+			}
+
+			function clearSearch() {
```

Comment:
This is used only in the 'close' button. I think we should remove this function, as well get rid of the 'close' button because it is superfluous (the user can always close the modal by clicking elsewhere).

Reply:
I'd remove them all, to keep the UI minimal.